iotdb-core/datanode/src/main/java/org/apache/iotdb/db/schemaengine/metric/SchemaEngineMemMetric.java

```diff
@@ -58,14 +58,18 @@ public void bindTo(AbstractMetricService metricService) {
         engineStatistics,
         ISchemaEngineStatistics::getTotalSeriesNumber,
         Tag.NAME.toString(),
-        TIME_SERES_CNT);
+        TIME_SERES_CNT,
+        Tag.DATABASE.toString(),
+        "total");
```

Comment:
So do we still need this field statistic? Instead, use grafana to calculate the value of all databases in the current node using sum

Reply:
Remove and we can use SCHEMA_REGION metric to statistic